test/test_jit.py

```diff
@@ -892,6 +892,53 @@ def forward(self, input):
         m_dropout.eval()
         self.assertEqual(dropout(input) + 1, m_dropout(input))
 
+    def test_nn_padding(self):
+        class Mod(nn.Module):
+            def __init__(self, padding):
+                super().__init__()
+                self.padding = padding
+
+            def forward(self, input):
+                return self.padding(input)
+
+        m = Mod(nn.ConstantPad1d(2, 3.5))
+        inp = torch.randn(1, 2, 4)
```

Comment:
was wondering if we can create list of tuple[nn.Module, Tensor] to simplify this. but we can probably do the refactoring in another pass

Reply:
Done!